modules/entity_browser/creating_browser_through_ui.md

```diff
@@ -1,7 +1,46 @@
-To create a new browser go to ```/admin/config/content/entity_browser``` and click on "Add Entity Browser". Follow the wizard according to your preferences through all the steps of the creation process.
 
-....
+### Creating a Browser with the multi-step Wizard
 
-This section of the Media Guide needs to be completed.
 
-See our [Contributing](../contributing.md) section of the Guide to find out how to get involved and help out.
\ No newline at end of file
+The Entity Browser module allows the creation of highly configurable browsers through the UI. Also, an overview of the existing browsers is available at the URL:
+
+``` /admin/config/content/entity_browser ```
+
+Make sure you have the permission ```administer entity browsers``` in order to access this page and create / edit the browsers.
+
+In order to create a new Entity Browser, click on **Add Entity Browser** or visit directly the URL ```/admin/config/content/entity_browser/add``` to access the multi-step wizard that will guide you through the process of creating / editing a browser.
```

Comment:
Maybe we could mention that this requires Ctools module (which is not a hard dependency for EB).